win/rfb_win32/SDisplay.cxx

```diff
@@ -312,7 +312,7 @@ void SDisplay::handleClipboardData(const char* data) {
 }
 
 
-void SDisplay::pointerEvent(const Point& pos, uint8_t buttonmask) {
+void SDisplay::pointerEvent(const Point& pos, uint16_t buttonmask) {
```

Comment:
This and `SInput` are for the Windows server, so they are misplaced in the client commit.

Perhaps move these changes, and the x0vncserver changes, to the Xvnc commit and simply have that change all servers?